### PR TITLE
Update README.md errors

### PR DESCRIPTION
**Pull request by rkc13**
This widget repo is awesome! Here is a couple recommendations to make the getting started code smoother.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ import 'package:flutter_circular_chart/flutter_circular_chart.dart';
 Create chart data entry objects:
 
 ```
+final GlobalKey<AnimatedCircularChartState> _chartKey;
```
Comment (xqwzts) [inline README.md:44]: Place this in its own code block with a leading comment explaining why we need it, something like:

    Create a [GlobalKey](https://docs.flutter.io/flutter/widgets/GlobalKey-class.html) to be able to access the chart and update its data:
    
    ```
    final GlobalKey<AnimatedCircularChartState> _chartKey;
    ```
    
    Create chart data entry objects:
    ...

**Comment by xqwzts** [comment]
@rkc13 Thanks for contributing!

That's a great suggestion to add the GlobalKey creation to the README, I've suggested a small modification in the diff comments, would you mind updating with that?

[If you'd rather not make the change that's fine, I can merge what you've sent now and apply the change myself later]

Thanks!

**Comment by rkc13** [comment]
Sounds great, you go ahead and merge it and apply it. I'm still new and figuring this out. Thanks for your positive response and thanks for your code!

**Comment by xqwzts** [comment]
Merged, thanks for contributing!

**Pull request merged**